src/components/FriendList/FriendSearch.js

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import SearchIcon from '@mui/icons-material/Search';
+import {InputAdornment} from "@mui/material";
+import TextField from '@mui/material/TextField';
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import AddIcon from "@mui/icons-material/Add";
+import {StudyRoomCard} from "../StudyRoom/CommonResources";
+import CheckIcon from '@mui/icons-material/Check';
+import axios from "axios";
+import GetAuthentication from "../Authentication/Authentification";
+
+
+export default function FriendSearch() {
+    const [added, setAdded] = React.useState(false);
+    const [found, setFound] = React.useState(false);
+    const [searchName, setSearchName] = React.useState(null);
+    const [friend, setFriend] = React.useState('');
+    const [list, setList] = React.useState('');
+    const [text, setText] = React.useState('');
+    const ownerEmail = GetAuthentication().email;
+    const ownerUsername = GetAuthentication().username;
+
+    React.useEffect(() => {
+        axios.get(`${process.env.REACT_APP_BASE_URL}friend/${ownerEmail}`)
+            .then(res => {
+                setList(res.data);
+            })
+            .catch(err => {
+                console.log('Error', err);
+            })
+
+    }, [text])
+
+    function resetState() {
+        setText('')
+        setFound(false);
+        setAdded(false);
+    }
+
+    //send a friend request
+    function handleAdd(e) {
+        e.preventDefault();
+        setAdded(true);
+        axios.post(`${process.env.REACT_APP_BASE_URL}friend/add`, {senderEmail: ownerEmail, receiverEmail: friend})
+            .then(() => {
+            })
+            .catch(err => {
+                console.log('Error:', err)
+            });
+        setSearchName("");
+    }
+
+    function handleSearch(e) {
+        e.preventDefault();
+        // can't search the owner
+        if (searchName === ownerEmail || searchName === ownerUsername) {
+            setText("this is the owner");
+        } else {
+            console.log(searchName);
```

Comment:
@kikkiii21 Please remove this console.log(...) before merging. Thank you!

Reply:
removed thanks, i missed it